articles/quickstart/spa/react/_includes/_centralized_login.md

```diff
@@ -130,22 +140,28 @@ Provide a component with controls for the user to log in and log out.
 ${snippet(meta.snippets.use)}
 
 ::: note
-This example uses Bootstrap styles, but that's unimportant. Use whichever style library you like, or don't use one at all.
+This example uses Bootstrap styles. You can use any style library you want, or not use one at all.
 :::
 
-The `onClick` events on the **Log In** and **Log Out** buttons make the appropriate calls to the `Auth` service to allow the user to log in and log out. Notice that these buttons are conditionally hidden and shown depending on whether or not the user is currently authenticated.
-
-When the **Log In** button is clicked, the user will be redirected to Auth0's hosted login page.
+Depending on whether the user is authenticated or not, they see the **Log Out** or **Log In** button. The `click` events on the buttons make calls to the `Auth` service to let the user log out or log in. When the user clicks the **Log In** button, they are redirected to the Auth0 hosted login page. 
 
 <%= include('../../_includes/_hosted_login_customization' }) %>
 
 ## Add a Callback Component
```

Comment:
In the Angular 2+ guide, the order of the 'Process authentication result' and 'Add a callback component' steps is reversed: 

Angular 2+:
Process result
Add a component

React: 
Add a component
Process result

Was that intentional? Or could we unify the order of those steps?